frontend/src/util/err.tsx

```diff
@@ -153,7 +153,7 @@ export const errorDisplayInfo = (error: unknown, i18n: i18n): ErrorDisplayInfo =
             };
         } else {
             return {
-                causes,
+                causes: Array.from(causes.keys()),
```

Comment:
```suggestion
                causes: Array.from(causes),
```

or even

```suggestion
                causes: [...causes],
```

?

Reply:
One might also question why we already convert here in the first place. :thinking: